### ISSUE TITLE
"Too many shared articles" error

### ISSUE THREAD

**Issue by sorburu** [ZachSaucier/Just-Read#176]
Sometimes i get this error message when clicking the JR extension, the icons appears huge and the text of the article without CSS.

I have only 3 articles shared. Below a screenshot:

https://s22.postimg.cc/l41y8wjb5/SNAG-0000.png

**Comment by ZachSaucier** [comment]
Interesting. So what's happening there is the stylesheet is completely not loading... 

I'm guessing this is the same as #174.

**Comment by ZachSaucier** [comment]
Is this happening on pages where you auto-run Just Read? And does the problem persist if you exit out of JR and start it again?

**Comment by sorburu** [comment]
Not using auto-run, and yes, exiting and trying again works fine. I have two CSS, default one and customized one, in case it helps.

**Comment by ZachSaucier** [comment]
Thanks for the info. I'll look into it ASAP.

**Comment by ZachSaucier** [comment]
If you can figure out a consistent way to reproduce this I'd greatly appreciate it. I've experienced it before but am having trouble reproducing it.

**Comment by sorburu** [comment]
Yes...i can't reproduce, exiting and trying again does not work always, right now it seems random to me :(

**Comment by ZachSaucier** [comment]
I am able to reproduce this error consistently on [this page](https://blog.ycombinator.com/learning-math-for-machine-learning/) by attempting to run Just Read before the page has fully loaded. That gives me hope for being able to solve this issue soon.
